<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文本编译器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res2.qrc">
    <normaloff>:/icons/图标/4515597.png</normaloff>:/icons/图标/4515597.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();  QTextCharFormat fmt=ui-&amp;gt;textEdit-&amp;gt;currentCharFormat();&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actFile_Open"/>
    <addaction name="actFile_Save"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actEdit_Cut"/>
    <addaction name="actEdit_Copy"/>
    <addaction name="actEdit_Paste"/>
    <addaction name="actEdit_Undo"/>
    <addaction name="actEdit_Redo"/>
    <addaction name="actEdit_Picture"/>
    <addaction name="actEdit_Table"/>
   </widget>
   <widget class="QMenu" name="menu_M">
    <property name="title">
     <string>格式(&amp;M)</string>
    </property>
    <addaction name="actFont_Bold"/>
    <addaction name="actFont_Italic"/>
    <addaction name="actFont_UnderLine"/>
    <addaction name="actFont_Color"/>
    <addaction name="actEdit_Left"/>
    <addaction name="actEdit_Right"/>
    <addaction name="actEdit_Center"/>
    <addaction name="actEdit_Side"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_M"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_Open"/>
   <addaction name="actFile_Save"/>
   <addaction name="separator"/>
   <addaction name="actEdit_Cut"/>
   <addaction name="actEdit_Copy"/>
   <addaction name="actEdit_Paste"/>
   <addaction name="actEdit_Undo"/>
   <addaction name="actEdit_Redo"/>
   <addaction name="separator"/>
   <addaction name="actFont_Bold"/>
   <addaction name="actFont_Italic"/>
   <addaction name="actFont_UnderLine"/>
   <addaction name="actFont_Color"/>
   <addaction name="separator"/>
   <addaction name="actEdit_Picture"/>
   <addaction name="actEdit_Table"/>
   <addaction name="actEdit_Left"/>
   <addaction name="actEdit_Right"/>
   <addaction name="actEdit_Center"/>
   <addaction name="actEdit_Side"/>
  </widget>
  <action name="actFile_Save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/3456668.png</normaloff>:/icon/图标/3456668.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actFile_Open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/2989332.png</normaloff>:/icon/图标/2989332.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actEdit_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1463072.png</normaloff>:/icon/图标/1463072.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actEdit_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/4515597.png</normaloff>:/icon/图标/4515597.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制到粘贴板</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actEdit_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334805.png</normaloff>:/icon/图标/1334805.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>从粘贴板粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actEdit_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334807.png</normaloff>:/icon/图标/1334807.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销上次编辑操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actEdit_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334808.png</normaloff>:/icon/图标/1334808.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做上次操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actFont_Bold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334820.png</normaloff>:/icon/图标/1334820.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体</string>
   </property>
  </action>
  <action name="actFont_Italic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334821.png</normaloff>:/icon/图标/1334821.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
  </action>
  <action name="actFont_UnderLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334822.png</normaloff>:/icon/图标/1334822.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
  </action>
  <action name="actFont_Color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/2983943.png</normaloff>:/icon/图标/2983943.png</iconset>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
   <property name="toolTip">
    <string>更改字体颜色</string>
   </property>
  </action>
  <action name="actEdit_Picture">
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1463060.png</normaloff>:/icon/图标/1463060.png</iconset>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
   <property name="toolTip">
    <string>插入图片</string>
   </property>
  </action>
  <action name="actEdit_Table">
   <property name="icon">
    <iconset resource="res2.qrc">
     <normaloff>:/icons/图标/5115909.png</normaloff>:/icons/图标/5115909.png</iconset>
   </property>
   <property name="text">
    <string>表格</string>
   </property>
   <property name="toolTip">
    <string>插入表格</string>
   </property>
  </action>
  <action name="actEdit_Left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334810.png</normaloff>:/icon/图标/1334810.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="toolTip">
    <string>文字左对其</string>
   </property>
  </action>
  <action name="actEdit_Right">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334812.png</normaloff>:/icon/图标/1334812.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="toolTip">
    <string>文字右对齐</string>
   </property>
  </action>
  <action name="actEdit_Center">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="photo.qrc">
     <normaloff>:/icon/图标/1334811.png</normaloff>:/icon/图标/1334811.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐</string>
   </property>
   <property name="toolTip">
    <string>文字居中对其</string>
   </property>
  </action>
  <action name="actEdit_Side">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res3.qrc">
     <normaloff>:/icons/图标3/7124176.png</normaloff>:/icons/图标3/7124176.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
   <property name="toolTip">
    <string>文字两端对齐</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res3.qrc"/>
  <include location="res2.qrc"/>
  <include location="photo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>textEdit</sender>
   <signal>redoAvailable(bool)</signal>
   <receiver>actEdit_Redo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit</sender>
   <signal>undoAvailable(bool)</signal>
   <receiver>actEdit_Undo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_Undo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_Redo</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_Paste</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_Cut</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actEdit_Copy</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
